core/genesis.go

```diff
@@ -316,7 +317,6 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, triedb *trie.Database, gen
 
 				// NOTE: kroma always post-regolith
 				zero := uint64(0)
-				config.BedrockBlock = new(big.Int).SetUint64(zero)
```

Comment:
Aren't there any problems if we have nil for the BedrockBlock? Like L420 in this file or L181 in eth/handler.go.

Reply:
As you said, it could be a problem.

Reply:
Hmm I wonder from LoadKromaChainConfig, isn't config.BedrockBlock always set to zero? Do we need this code?

Reply:
Needed. If it is not the chain id defined in LoadConfig, the BedrockBlock will be nil.